databases/week1/DatabaseWeek1.sql

```diff
@@ -0,0 +1,48 @@
+USE database_week1;
+
+-- Find out how many tasks are in the task table
+SELECT COUNT(id)
+FROM task;
+      
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT due_date
+FROM task
+WHERE due_date = null;
```

Comment:
Here we need to know how many tasks where they don't have a valid due_date, we need to count
```
SELECT COUNT(*) FROM task WHERE due_date IS NULL;
--Alternative
SELECT task.id,task.title,task.due_date FROM task
WHERE due_date is NULL;
```

Reply:
ok